05.wc/wc.rb

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'optparse'
+
+def main
+  opt = parse_options
```

Comment:
```suggestion
  options = parse_options
```

名前をあわせてあげるとよいかも
https://bootcamp.fjord.jp/pages/readable-variable-name

Reply:
`options`に統一しました。